### ISSUE TITLE
Enable SourceLink to allow easier debugging (Step into the code of this library)

### ISSUE THREAD

**Issue by MagicAndre1981** [contre/Windows-API-Code-Pack-1.1#17]
Visual Studio 2017 supports a new debug option called [SourceLink](https://github.com/dotnet/designs/blob/master/accepted/diagnostics/source-link.md#source-link) which adds a JSON file to the PDB (which is now part of the NuGet package) which tells the Visual Studio Debugger to fetch the .cs files from a repository like for JSON.net based on commit hash [7217c484e9705b5e76585c8b7fcd489c8e021c23](https://github.com/JamesNK/Newtonsoft.Json/commit/7217c484e9705b5e76585c8b7fcd489c8e021c23) 

```
<SourceLink>
    <Conversion FilePath="/_/*" Uri="https://raw.githubusercontent.com/JamesNK/Newtonsoft.Json/7217c484e9705b5e76585c8b7fcd489c8e021c23/*" />
</SourceLink>
```

See the Json.net [pull request](https://github.com/JamesNK/Newtonsoft.Json/pull/1746) how to do it.

**Comment by contre** [comment]
I've not published new nuget packages but I believe I've now enabled this functionality

**Comment by MagicAndre1981** [comment]
ok, I see that you converted the project to new SDK style, with packages reference include, but still have old packages.config present. Delete old files. I'll test it when new nugets are published

**Comment by MagicAndre1981** [comment]
the PDBs/snupkg file is missing on nuget, so sourcelink can't work. @contre

**Comment by contre** [comment]
how bout now? I forgot I was supposed to package and push them so I only just uploaded them.

**Comment by MagicAndre1981** [comment]
> how bout now? 

yes, it works now as expected, the symbols are loaded and I can step into the code:

![image](https://user-images.githubusercontent.com/8274816/76187318-50380b80-61d5-11ea-9a83-d5365ea4c06a.png)
